pkgs/tools/graphics/gmic-qt/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv, fetchurl, fetchFromGitHub, cmake, ninja, pkgconfig, qt5
```

Comment:
according to https://nixos.org/nixpkgs/manual/#sec-language-qt you should not use `qt5` as an argument but use `libsForQt5.callPackage` (or maybe a plugin qualifies as a library ?)

Reply:
Agreed.

Reply:
well, gmic_krita_qt depends on qt5 as its gui framework...

Reply:
True, but if you use `libsForQt5.callPackage` in all-packages.nix the you reference dependencies directly instead of using the qt5 set. For example `{ qtbase }` instead of `{ qt5 }`.

Reply:
This ensures that all dependencies have the same qt5 version.

Reply:
i changed it in the latest commit